website/screens/principles/localization/LocalizationPage.tsx

```diff
@@ -193,6 +193,49 @@ const sections = [
           </DxcTable>
         ),
       },
+      {
+        title: "calendar",
+        content: (
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Label Name</th>
+                <th>Default value</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <Code>daysShort</Code>
+                </td>
+                <td>[Mo, Tu, We, Th, Fr, Sa, Su]</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>months</Code>
+                </td>
+                <td>
+                  [ January, February, March, April, May, June, July, August,
+                  September, October, November, December ]
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>previousMonthTitle</Code>
+                </td>
+                <td>previous month</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>nextMonthTitle</Code>
+                </td>
+                <td>next month</td>
+              </tr>
+            </tbody>
+          </DxcTable>
+        ),
+      },
```

Comment:
`days` and `openCalendar` are missing.

Reply:
These two do not appear in the localization because they are not used for translation.

`days` is simply not used so I will remove it from `variables`.
In the case of `openCalendar` we tried to implement it for the calendar action but due to limitations with the text-input, we were not able to use the translation for the action title without making changes in the text-input.

Reply:
So it makes no sense to have it on variables for the moment no? We can remove `openCalendar`.

Reply:
done